app/services/email_service.rb

```diff
@@ -10,8 +10,7 @@ def initialize(message:, allocation:, pom_nomis_id:)
     @allocation = allocation
 
     @offender = OffenderService.get_offender(@allocation[:nomis_offender_id])
-    @pom = PrisonOffenderManagerService.get_pom(
-      @allocation.prison,
+    @pom = Prison.new(@allocation.prison).get_pom(
```

Comment:
Can we pass in the prison here?

Reply:
again, this is a design issue. It's just more obvious now we have a data layer